### PR TITLE
Add the option to install the map with Overlay with (and without) markers

### PR DESCRIPTION
**Pull request by matheus-rib**

### CODE REVIEW

File: install-tibia-maps [hunk 1]
```diff
@@ -2,13 +2,21 @@
...
 mkdir -p "${TIBIA_DIR}";
-if [ "${1}" = '--no-markers' ]; then
-	echo 'Downloading & extracting `minimap-without-markers.zip`…';
-	url='https://tibiamaps.io/downloads/minimap-without-markers';
-else
-	echo 'Downloading & extracting `minimap-with-markers.zip`…';
-	url='https://tibiamaps.io/downloads/minimap-with-markers';
-fi;
+echo $1
+case $1 in
```
Comment (mathiasbynens) [inline install-tibia-maps]:
```suggestion
case "${1}" in
```

File: install-tibia-maps [hunk 2]
```diff
@@ -2,13 +2,21 @@
...
+	--grid)
+		echo 'Downloading & extracting `minimap-with-grid-overlay-and-markers.zip`…';
+		url='https://tibiamaps.io/downloads/minimap-with-grid-overlay-and-markers';;
```
Comment (mathiasbynens) [inline install-tibia-maps]:
```suggestion
		url='https://tibiamaps.io/downloads/minimap-with-grid-overlay-and-markers';
```

File: install-tibia-maps [hunk 3]
```diff
@@ -2,13 +2,21 @@
...
+	--grid-no-markers)
+		echo 'Downloading & extracting `minimap-with-grid-overlay-without-markers.zip`…';
+		url='https://tibiamaps.io/downloads/minimap-with-grid-overlay-without-markers';;
```
Comment (mathiasbynens) [inline install-tibia-maps]:
```suggestion
		url='https://tibiamaps.io/downloads/minimap-with-grid-overlay-without-markers';
```

File: install-tibia-maps [hunk 4]
```diff
@@ -2,13 +2,21 @@
...
+	--no-markers)
+		echo 'Downloading & extracting `minimap-without-markers.zip`…';
+		url='https://tibiamaps.io/downloads/minimap-without-markers';;
```
Comment (mathiasbynens) [inline install-tibia-maps]:
```suggestion
		url='https://tibiamaps.io/downloads/minimap-without-markers';
```

File: install-tibia-maps [hunk 5]
```diff
@@ -2,13 +2,21 @@
...
+	*)
+		echo 'Downloading & extracting `minimap-without-markers.zip`…';
+		url='https://tibiamaps.io/downloads/minimap-without-markers';;
```
Comment (mathiasbynens) [inline install-tibia-maps]:
```suggestion
		url='https://tibiamaps.io/downloads/minimap-without-markers';
```

File: install-tibia-maps [hunk 6]
```diff
@@ -2,13 +2,21 @@
...
+esac
```
Comment (mathiasbynens) [inline install-tibia-maps]:
```suggestion
esac;
```

**Review by mathiasbynens — COMMENTED** [review]
Thanks! Looks good, just a few nits.

File: install-tibia-maps [hunk 7]
```diff
@@ -2,13 +2,20 @@
...
+		url='https://tibiamaps.io/downloads/minimap-with-grid-overlay-and-markers';
+	--grid-no-markers)
+		echo 'Downloading & extracting `minimap-with-grid-overlay-without-markers.zip`…';
+		url='https://tibiamaps.io/downloads/minimap-with-grid-overlay-without-markers';
+	--no-markers)
+		echo 'Downloading & extracting `minimap-without-markers.zip`…';
+		url='https://tibiamaps.io/downloads/minimap-without-markers';
+	*)
+		echo 'Downloading & extracting `minimap-without-markers.zip`…';
+		url='https://tibiamaps.io/downloads/minimap-without-markers';
```
Comment (mathiasbynens) [inline install-tibia-maps]:
```suggestion
		echo 'Downloading & extracting `minimap-with-markers.zip`…';
		url='https://tibiamaps.io/downloads/minimap-with-markers';
```

**Review by mathiasbynens — CHANGES_REQUESTED** [review]

**Comment by matheus-rib** [comment]
Sorry, it was my first time facing a shell script and yeah, i forgot that echo there, gonna correct all the appointments, thanks

File: install-tibia-maps [hunk 8]
```diff
@@ -2,13 +2,24 @@
...
-	url='https://tibiamaps.io/downloads/minimap-without-markers';
-else
-	echo 'Downloading & extracting `minimap-with-markers.zip`…';
-	url='https://tibiamaps.io/downloads/minimap-with-markers';
-fi;
+case "${1}" in
+	--grid)
+		echo 'Downloading & extracting `minimap-with-grid-overlay-and-markers.zip`…';
+		url='https://tibiamaps.io/downloads/minimap-with-grid-overlay-and-markers';
+		break;;
```
Comment (mathiasbynens) [inline install-tibia-maps]:
```suggestion
		break;
		;;
```

File: install-tibia-maps [hunk 9]
```diff
@@ -2,13 +2,24 @@
...
+	--grid-no-markers)
+		echo 'Downloading & extracting `minimap-with-grid-overlay-without-markers.zip`…';
+		url='https://tibiamaps.io/downloads/minimap-with-grid-overlay-without-markers';
+		break;;
```
Comment (mathiasbynens) [inline install-tibia-maps]:
```suggestion
		break;
		;;
```

File: install-tibia-maps [hunk 10]
```diff
@@ -2,13 +2,24 @@
...
+	--no-markers)
+		echo 'Downloading & extracting `minimap-without-markers.zip`…';
+		url='https://tibiamaps.io/downloads/minimap-without-markers';
+		break;;
```
Comment (mathiasbynens) [inline install-tibia-maps]:
```suggestion
		break;
		;;
```

File: install-tibia-maps [hunk 11]
```diff
@@ -2,13 +2,24 @@
...
+	*)
+		echo 'Downloading & extracting `minimap-with-markers.zip`…';
+		url='https://tibiamaps.io/downloads/minimap-with-markers';
+		break;;
```
Comment (mathiasbynens) [inline install-tibia-maps]:
```suggestion
		break;
		;;
```

**Review by mathiasbynens — APPROVED** [review]

**Comment by mathiasbynens** [comment]
Thanks! Nice work.

**Pull request merged**